src/t_stream.c

```diff
@@ -2698,9 +2703,18 @@ void xautoclaimCommand(client *c) {
         return;
     }
 
-    if (getLongLongFromObjectOrReply(c,c->argv[6],&count,NULL) != C_OK)
-        return;
-    if (count < 0) count = 0;
+    if (c->argc == 8) {
+        if (strcasecmp(c->argv[6]->ptr,"count")) {
```

Comment:
this code is really explicit. it may be ok for now, but also we can convert it to a standard way of parsing optional arguments, like the one we have in SORT
WDYT?
then we don't need the `if` on argc above (the final `else` will handle it)